docs/src/index.md

```diff
@@ -4,7 +4,33 @@ CurrentModule = Taxonomy
 
 # Taxonomy
 
-Documentation for [Taxonomy](https://github.com/StructuralEquationModels/Taxonomy.jl).
+`Taxonomy.jl` is work in progress but here is our vision about what it is supposed to do when finished:
+
+## What?
+
+Taxonomy.jl aims to serve as a comprehensive database of structural equation models (SEMs) that can be used to infer distributions of both structures and parameters.
+This will greatly facilitate simulation studies that accurately reflect real-world conditions.
+Additionally, Taxonomy.jl will provide a user-friendly interface for researchers to easily sample these parameters for use in their own simulation studies.
+
+### End product
+
+A Julia package that enables filtering a taxonomy database and construct samplers for structure and parameters.
+These samplers can quickly be turned into models for [StructuralEquationModels.jl](github.com/StructuralEquationModels/StructuralEquationModels.jl).
+
+
+## So what?
+
+Simulations are only in so far helpful as they reflect some (simplified) aspect of reality.
```

Comment:
Simulations are only helpful in so far as they reflect some (simplified) aspects of reality.

Reply:
```suggestion
Simulations are only helpful in so far as they reflect some (simplified) aspects of reality.
```